test/python/test_qasm_sympy_simulator.py

```diff
@@ -61,7 +61,7 @@ def setUp(self):
                          }
                      ]}
         self.q_job = QuantumJob(self.qobj,
-                                backend='local_sympy_qasm_simulator',
+                                backend=SympyQasmSimulator,
```

Comment:
I guess you meant: `SympyQasmSimulator()` ?

Reply:
yep, thanks!